nosqlbench/http-jsonapi-crud-dataset.yaml

```diff
@@ -32,6 +35,7 @@ bindings:
 
```

Comment:
let's update the `weighted_hosts: WeightedStrings('<<jsonapi_host:jsonapi>>')` to what I had in other test:

```
weighted_hosts: WeightedStrings('<<jsonapi_host:<<stargate_host:localhost>>>>')
```